### ISSUE TITLE
Integrate cuzzle in the profiler

### ISSUE THREAD

**Issue by iamluc** [csarrazi/CsaGuzzleBundle#82]
Quoting https://github.com/namshi/cuzzle/
```
This library let's you dump a Guzzle request to a cURL command for debug and log purpose. 
```

But sadly, Guzzle 6 is not yet supported. See https://github.com/namshi/cuzzle/issues/8

**Comment by csarrazi** [comment]
I'm not sure this library is still maintained. Ping @namshi ?

**Comment by cordoval** [comment]
is not evidently

**Comment by csarrazi** [comment]
I did a PR to update namshi/cuzzle to use guzzle 6.x. See namshi/cuzzle#10

**Comment by csarrazi** [comment]
Cuzzle has been updated for Guzzle 6.0. @iamluc Want to implement this? :)

**Comment by iamluc** [comment]
Hey ! 
I will have a look at it this week-end :-)

Thanks for your work on the library.

**Comment by csarrazi** [comment]
You're welcome! :)

**Comment by csarrazi** [comment]
Implemented in #145
